sql/pgtap.sql.in

```diff
@@ -869,11 +869,11 @@ BEGIN
     SELECT GREATEST((return_percent * iterations)::int, 1) INTO limit_it;
 
     FOR a_time IN SELECT times[i]
-		  FROM generate_series(array_lower(times, 1), array_upper(times, 1)) i
+                  FROM generate_series(array_lower(times, 1), array_upper(times, 1)) i
```

Comment:
I didn't realize there were some wayward tabs. I removed them in 0c0197a.

Reply:
Ok